data/json/npcs/refugee_center/NPC_Aleesha_Seward.json

```diff
@@ -157,7 +157,7 @@
     "id": "TALK_REFUGEE_Aleesha_2_veryearly",
     "dynamic_line": "I don't know what's up.  I'm not sure what we've even doing here.  They say we're supposed to wait until we can be moved to the shelter downstairs, but we've been here days and there's no word on how long we'll be waiting.  It's all so stupid, and nobody can tell me anything.",
     "responses": [
-      { "text": "What's your story?  How did you get here?", "topic": "TALK_REFUGEE_Aleesha_Story1" },
+      { "text": "What's your story?  How did you get here?", "topic": "TALK_REFUGEE_Aleesha_/1" },
```

Comment:
It looks like you introduced a new typo here? Unless there's a syntax I'm unaware of.